### PR TITLE
Update rust import style setting

### PR DESCRIPTION
**Pull request by 6367f766**
## Proposed changes

Added a rustfmt.toml file and changed all imports according to this file. Note nightly toolchain is needed.

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement (general improvements like code refactoring that doesn't explicitly fix a bug or add any new functionality)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Paste Link to the issue
- #1660

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [x] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: crates/common/certificate/src/parse_root_certificate.rs [hunk 1]
```diff
@@ -1,6 +1,14 @@
-use rustls::{Certificate, ClientConfig, PrivateKey, RootCertStore};
-use rustls_pemfile::{certs, pkcs8_private_keys, rsa_private_keys};
-use std::{fs, fs::File, io::BufReader, path::PathBuf};
+use rustls::Certificate;
+use rustls::ClientConfig;
```
Comment (PradeepKiruvale) [inline crates/common/certificate/src/parse_root_certificate.rs:2]: If I remember correctly, Didier was not for this kind of import but for the old type, grouped imports. Need to confirm with @didier-wenzek

Comment (reubenmiller) [reply crates/common/certificate/src/parse_root_certificate.rs:2]: Yes we can wait for confirmation from @didier-wenzek himself but my notes were for this method solely due to the benefits of cleaner git diffs (to assist code reviews)

Comment (rina23q) [reply crates/common/certificate/src/parse_root_certificate.rs:2]: We agreed to use [`Item`](https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=#Item%5C%3A) and @didier-wenzek was also in the call.

**Review by rina23q — APPROVED** [review]
LGTM

File: crates/common/certificate/src/parse_root_certificate.rs (hunk 1 above)
Comment (didier-wenzek) [reply crates/common/certificate/src/parse_root_certificate.rs:2]: Item is perfect!

**Pull request merged**